### PR TITLE
Profile Section Documented

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :nut_and_bolt: Description: What code changed, and why?

The Profile section of the Chef Automate UI has been documented.

### :chains: Related Resources

- https://github.com/chef/chef-web-docs/issues/3353

### :+1: Definition of Done

A new page has been added for the section under Settings.

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+You can update the display name for your Chef Automate in the bar.
+
+{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}
+
+The Display Name bar should not be kept empty or else it throws error.
+
+{{< figure src="/images/automate/error-in-display-name-bar.png" alt="Error in Display Name">}}
+
+### Update the Date/Time Format
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:33]:
```suggestion
### Change the Date/Time Format
```

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+You can select the date and time format amongst the five formats provided in Chef Automate.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:35]:
```suggestion
You can select one of five date and time formats.
```

Are we changing the time zone or just the formatting? It isn't clear to me what the user can do.

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: components/docs-chef-io/content/automate/chef_automate_profile.md (hunk 2 above)
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:35]: Also, it doesn't look like you removed the old section from the docs.

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
++++
+title = "Profile"
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:2]: Profile is something which we can see in our Chef Automate, but yes totally agreed that 'User Profile' would be more findable

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    parent = "automate/settings"
+    identifier = "automate/settings/chef_automate_profile.md Profile"
+    weight = 120
++++
+
+## Overview
+
+The Chef Automate **Profile** help you set your personal information about the entities. You can set the _Display Name_, _Time Format_ and reset your _Password_.
+
+{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:21]: We do not have a specific name for the dropdown but yes we can add the full screen and outline this section

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+### Update the Display Name
+
+You can update the display name for your Chef Automate in the bar.
+
+{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}
+
+The Display Name bar should not be kept empty or else it throws error.
+
+{{< figure src="/images/automate/error-in-display-name-bar.png" alt="Error in Display Name">}}
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:33]: It gets updated at the same moment.

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+### Update the Date/Time Format
+
+You can select the date and time format amongst the five formats provided in Chef Automate.
+
+{{< figure src="/images/automate/date-time-format-in-profile.png" alt="Date/Time Format">}}
+
+Select the date/time format and save to get the changes reflected in the interface.
```
[the remark this answers is not in the document]
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:41]: It gets overwrite. We will be able to see the changed time format

File: components/docs-chef-io/content/automate/chef_automate_profile.md (hunk 2 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_automate_profile.md]: We can just change the formatting

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    weight = 120
++++
+
+## Overview
+
+Chef Automate users can manage their own display name, date/time format, and password. Select the user icon in the top navigation bar, and then select **Profile** from the drop-down menu.
+
+{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}
+
+Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:21]:
```suggestion
Select **Profile** to view your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
```
```suggestion
Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
```

Comment (kagarmoe) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:21]: Could you disambiguate between the `username` and the `Display Name`.  I don't know the difference between the two from this doc.

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+### Change a Display Name
+
+You can change the display name for your Chef Automate in the bar.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:25]:
```suggestion
Change your display name by entering your preferred name.
```

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}
+
+{{< note >}}
+The **Display Name** gets updated as soon as you change the name in the bar.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:30]: Don't use a note for this. Put the content above the image. 

Also, I would think that the display name changes when the user hits `save`?

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+{{< /note >}}
+
+The Display Name bar should not be kept empty or else it throws error.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:33]:
```suggestion
Every user needs a display name. An empty display name causes an error.
```
```suggestion
The Display Name bar should not be kept empty or else it throws error.
```

File: components/docs-chef-io/content/automate/chef_automate_profile.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:35]: That's important. Please list all of the things that get updated.

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+### Change the Date/Time Format
+
+You can select one of five date and time formats.
+
+{{< figure src="/images/automate/date-time-format-in-profile.png" alt="Date/Time Format">}}
+
+Select the date/time format and save to get the changes reflected in the interface.
+
+{{< note >}}
+The **Date/Time Format** gets updated as soon as you select a new time format from the drop down.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:46]: Again, wouldn't the user need to `save` first?

File: components/docs-chef-io/content/automate/chef_automate_profile.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:43]: That's important--and something that the user should know!

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+{{< /note >}}
+
+### Reset Password
+
+You can reset the password of Chef Automate login from the **Reset Password** tab in Profile.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:51]:
```suggestion
Reset your password of Chef Automate login from the **Reset Password** tab in Profile.
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: components/docs-chef-io/content/automate/chef_automate_profile.md (hunk 7 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:21]: Yes!
`username` is the login name which we need to login into our account, whereas display name is the name displayed in profiles.

File: components/docs-chef-io/content/automate/chef_automate_profile.md (hunk 9 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:30]: Yes!
Will update both the pointers

File: components/docs-chef-io/content/automate/chef_automate_profile.md (hunk 11 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:46]: Yes!
Have updated the same

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+Chef Automate users can manage their display name, date/time format, and password. Select the user icon in the top navigation bar, and select **Profile** from the drop-down menu.
+
+{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}
+
+Select **Profile** to view the details. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you edit your password.
+
+### Change a Display Name
+
+Change your display name by entering your preferred name. The display name gets updated as soon as you change the name in the bar and click **Save**.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:25]:
```suggestion
Change your display name by entering your preferred name. Select **Save** to update your name.
```

File: components/docs-chef-io/content/automate/chef_automate_profile.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+
+{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}
+
+Every user needs a display name. An empty display name causes an error.
+
+{{< figure src="/images/automate/error-in-display-name-bar.png" alt="Error in Display Name">}}
+
+### Change the Date/Time Format
+
+You can select one of five date and time formats from the drop-down. The date/time format will get updated as soon as you choose anyone and click **Save**.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_automate_profile.md:35]:
```suggestion
Select one of the five date and time formats from the drop-down menu. Select **Save** to change the appearance of the date and time across Chef Automate.
```

File: components/docs-chef-io/content/automate/chef_automate_profile.md (hunk 13 above)
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/chef_automate_profile.md:25]: I'm concerned that my phrase `across Chef Automate.` is too colloquial. Does it make sense to you?

**Review by kagarmoe — CHANGES_REQUESTED** [review]
You used `click`?!

**Review by kagarmoe — APPROVED** [review]
tenor-181998128

**Review by kalroy — APPROVED** [review]

**Pull request merged**